### PR TITLE
a more cleared and simplified delete workspace in terms of code

### LINKED ISSUE #23: potientially redudant code in code-4.3 container_process.go
//Delete the AUFS filesystem while container exit
func DeleteWorkSpace(rootURL string, mntURL string, volume string){
	/f(volume != ""){
		volumeURLs := volumeUrlExtract(volume)
		length := len(volumeURLs)
		if(length == 2 && volumeURLs[0] != "" && volumeURLs[1] !=""){
			DeleteMountPointWithVolume(rootURL, mntURL, volumeURLs)
		}else{
			DeleteMountPoint(rootURL, mntURL)
		}
	}else {
		DeleteMountPoint(rootURL, mntURL)
	}
	DeleteWriteLayer(rootURL)
}

seems can be done by
if (volume) {
   .. umount volume
}
.. umount container mnt
delete write layer

basically the if else clause have room be simplified

### PR DESCRIPTION
**Pull request by s3341458**
close #23 

a more simplified code reduce some duplicate code and if else clause

### CODE REVIEW

File: container/volume.go [hunk 1]
```diff
@@ -123,23 +120,13 @@ func DeleteMountPoint(containerName string) error {
...
 		log.Errorf("Umount volume %s failed. %v", containerUrl, err)
 		return err
 	}
-
-	if _, err := exec.Command("umount", mntURL).CombinedOutput(); err != nil {
-		log.Errorf("Umount mountpoint %s failed. %v", mntURL, err)
-		return err
-	}
-
-	if err := os.RemoveAll(mntURL); err != nil {
```
Comment (xianlubird) [inline container/volume.go]: 这里会不会留下mntURL文件夹？

Comment (s3341458) [reply container/volume.go]: DeleteMountPoint(containerName) will always been called, so I assume should be fine.
the purpose is to decouple rm volume with rm container.

Comment (s3341458) [reply container/volume.go]: 就是DeleteMountPointWithVolume 和 DeleteMountPoint 两个函数有一部分重合了，属于代码冗余。
改编以后DeleteVolume确实不会删除mntURL, 但是DeleteMountPoint会删除mntURL.

改编以前大概是这样的：
if volume specified:
   remove volume and mnt(in one function)
else:
   remove mnt(another function)

改编之后:
if volume specified:
   remove volume
remove mnt

对了，linux上什么中文输入法比较好用呀，我打中文好累呀。

Comment (BSWANG) [reply container/volume.go]: 之前用linux都是在里面装个虚拟机去聊天和访问网页的。。。。

Comment (s3341458) [reply container/volume.go]: 现在linux的browser 还是很不错的

Comment (xianlubird) [reply container/volume.go]: 👍

**Pull request merged**